build.gradle

```diff
@@ -30,7 +30,7 @@ buildscript {
             // kotlin libraries. Make sure to keep this dependency in line with the kotlin version used.
             "kotlinCoroutinesCore"       : "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.1",
 
-            "browserSwitch"              : "com.braintreepayments.api:browser-switch:3.0.0-beta1-SNAPSHOT",
```

Comment:
I think we can also remove the maven URL for snapshots in the `repositories` section below

Reply:
Good catch! Just removed it.